src/macaron/resources/.gitignore

```diff
@@ -7,3 +7,6 @@ gradle-*
 
 # Ignore Gradle build output directory
 build
+
+# Ignore downloaded schemastore files
+schemastore
```

Comment:
I think you needed to include the path to schemastore because it's ignored here?

Reply:
Without the gitignore, flit won't let you build the package at all, it aborts if there are untracked files.